cypress/integration/brewery/ScenarioParametersValidation.spec.js

```diff
@@ -17,23 +18,75 @@ describe('scenario parameters inputs validation', () => {
       UPDATE_SCENARIO: true,
     });
     Login.login();
+    stub.setSolutions(SOLUTIONS);
+    stub.setScenarios(SCENARIOS);
   });
-  it('can clear number input, checks error message and Save and Launch buttons', () => {
-    ScenarioParameters.expandParametersAccordion();
-    ScenarioParameters.getInputValue(BreweryParameters.getCurrencyValueInput()).as('currency-value');
-    BreweryParameters.getCurrencyValueInput().click().clear();
-    BreweryParameters.getCurrencyValueInput().blur();
-    BreweryParameters.getCurrencyValueInput().should('value', '');
-    BreweryParameters.getCurrencyValueHelperText().should('be.visible');
-    ScenarioParameters.getSaveButton().should('be.disabled');
-    ScenarioParameters.getLaunchButton().should('be.disabled');
-    BreweryParameters.switchToDatasetPartsTab();
-    ScenarioParameters.getSaveButton().should('be.disabled');
-    ScenarioParameters.getLaunchButton().should('be.disabled');
-    ScenarioParameters.discard();
-    BreweryParameters.switchToBasicTypesTab();
-    cy.get('@currency-value').then((input) => {
-      BreweryParameters.getCurrencyValueInput().should('value', input);
-    });
+
+  beforeEach(() => {
+    Login.relogin();
   });
+
+  it(
+    'clears number input, checks required value error message, Save and Launch buttons, ' +
+      'changes tab and checks that Save and Launch buttons are still disabled',
+    () => {
+      ScenarioParameters.expandParametersAccordion();
+      ScenarioParameters.getInputValue(BreweryParameters.getRestockInput()).as('restock');
+      BreweryParameters.getRestockInput().click().clear();
+      BreweryParameters.getRestockInput().blur();
```

Comment:
Is _blur_ relay useful ?

Reply:
I added this blur + further check of value (empty string) to be sure that we don't have a "NaN" displayed in the TextField :slightly_smiling_face: